platformio.ini

```diff
@@ -12,7 +12,7 @@
 ;default_envs = travis_esp8266, travis_esp32
 
 # Release binaries
-default_envs = nodemcuv2, esp01_1m_full, esp32dev, esp32_eth
+default_envs = esp32dev
```

Comment:
Do not change platformio.ini except when adding new environments.
Please create platformio_override.ini and put your custom environment there.